addons/web/static/tests/core/dom_tests.js

```diff
@@ -0,0 +1,130 @@
+odoo.define('web.dom_tests', function (require) {
+"use strict";
+
+var dom = require('web.dom');
+
+QUnit.module('core', {}, function () {
+QUnit.module('dom', {}, function () {
+
+    QUnit.module('autoresize');
+
+    QUnit.test('autoresize (border-box): no padding + no border', function (assert) {
+        assert.expect(3);
+        var $textarea = $('<textarea>');
```

Comment:
i think that you can extract lines 13->21 in a helper function in the top of the file. Something like

```
function prepareTextArea(options) {
   var $textarea = $('<textarea>');
   $textarea.css('box-sizing', 'border-box');
  $textarea.css(options);
  $textarea.appendTo($('#qunit-fixture'));
  dom.autoresize($textarea, { min_height: 1 });
  return $textarea;
}

```

Reply:
and reuse it in next tests